includes/wc-product-functions.php

```diff
@@ -200,6 +200,40 @@ function wc_get_featured_product_ids() {
 	return $featured_product_ids;
 }
 
+/**
+ * Populate a batch of rating count lookup table data for products.
+ *
+ * @since 4.0.1
+ * @param int $product Product Object.
+ */
+function wc_get_parent_grouped_id( $product ) {
```

Comment:
This function name doesn't sounds accurate, probably better something like `wc_get_product_parent_groups( $product )`.